app/scripts/directives/oscFileInput.js

```diff
@@ -188,6 +188,17 @@ angular.module('openshiftConsole')
         }
 
         function addFile(file) {
+          // If the file is larger than 5 MiB, don't try to upload. Note that secrets and config
+          // maps have an even smaller limit (1 MB).
+          if (file.size > (5 * 1024 * 1024)) {
```

Comment:
Only thought would be to take this `(5 * 1024 * 1024)` and put it in a `.constant("FILE_UPLOAD_MAX",5 * 1024 * 1024)` in `app.js`.  Otherwise lgtm.

Reply:
Since we're only using this in one place, I think it's better to keep it near where it's used.

Reply:
Works for me.